tests/test_browser.py

```diff
@@ -1817,8 +1823,8 @@ def test_gles2_emulation(self):
                        path_from_root('tests', 'glbook', 'Common', 'esShader.c'),
                        path_from_root('tests', 'glbook', 'Common', 'esShapes.c'),
                        path_from_root('tests', 'glbook', 'Common', 'esTransform.c'),
-                       '-s', 'FULL_ES2=1', '-lGL', '-lEGL', '-lX11',
-                       '--preload-file', 'basemap.tga', '--preload-file', 'lightmap.tga', '--preload-file', 'smoke.tga'])
+                       '-lGL', '-lEGL', '-lX11',
+                       '--preload-file', 'basemap.tga', '--preload-file', 'lightmap.tga', '--preload-file', 'smoke.tga'] + args)
```

Comment:
I think `-s FULL_ES2=1` a few lines below may probably be removed now, as well as the comment.

Reply:
@hujiajie do you mean in the next test, `test_clientside_vertex_arrays_es3`? (so unrelated to this PR?)

Reply:
Oh I see that test has both flags atm, good point! Fixing.